service/controller/control-plane/resource.go

```diff
@@ -131,6 +136,35 @@ func newResources(config resourcesConfig) ([]resource.Interface, error) {
 		}
 	}
 
+	var serviceResource resource.Interface
+	{
+		c := service.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+		}
+
+		serviceResource, err = service.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```

Comment:
What is this for? There's already a `prometheus-operated` service created by prometheus-operator,

Reply:
This service is for the statefulset to actually work and does not have a cluster IP so we cannot use it for the ingress

Reply:
This works I did it there https://github.com/giantswarm/giantswarm/issues/11524#issuecomment-696832297

Reply:
Cool, so we can do without, but if we manage it, we can add annotations to it which we cannot on the statefulset service